packages/react/changelog.yaml

```
@@ -0,0 +1,11 @@
+unreleased:
+  type: minor
+
+  changed:
+    - summary: Initialize @cerbos/react, A collection of React hooks for interacting with Cerbos policy decision points.
+      pull: 876
+
+references:
+  nodejs-eol: https://github.com/nodejs/release#end-of-life-releases
+
+initialCommit: a3dcb572e0eeff4c4d86c9cc66c1d0c7e59c4853
```

`initialCommit` is maybe not the right name here in retrospect. It's the last commit before the package was introduced (i.e. where to start diffing from for the first release). I'll avoid merging anything else before this one now, so that we don't have to keep updating this.

```suggestion
initialCommit: 68337848cb3f987627da7381653d82c6d4e368a5
```